src/main/java/com/voicepin/flow/client/FlowClient.java

```diff
@@ -1,45 +1,176 @@
 package com.voicepin.flow.client;
 
-
-import com.voicepin.flow.client.calls.*;
+import com.voicepin.flow.client.calls.AddVoiceprintCall;
+import com.voicepin.flow.client.calls.Call;
+import com.voicepin.flow.client.calls.EnrollCall;
+import com.voicepin.flow.client.calls.GetVoiceprintCall;
+import com.voicepin.flow.client.calls.VerifyInitCall;
 import com.voicepin.flow.client.exception.FlowClientException;
-import com.voicepin.flow.client.request.*;
-import com.voicepin.flow.client.result.*;
+import com.voicepin.flow.client.request.AddVoiceprintRequest;
+import com.voicepin.flow.client.request.EnrollRequest;
+import com.voicepin.flow.client.request.GetVoiceprintRequest;
+import com.voicepin.flow.client.request.VerifyInitRequest;
+import com.voicepin.flow.client.request.VerifyRequest;
+import com.voicepin.flow.client.result.AddVoiceprintResult;
+import com.voicepin.flow.client.result.EnrollResult;
+import com.voicepin.flow.client.result.GetVoiceprintResult;
+import com.voicepin.flow.client.result.VerifyInitResult;
+import com.voicepin.flow.client.ssl.PinnedCertificateStrategy;
+import com.voicepin.flow.client.ssl.TrustedCertificateStrategy;
+import com.voicepin.flow.client.ssl.CertificateStrategy;
+import com.voicepin.flow.client.ssl.AnyCertificateStrategy;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * @author kodrzywolek
+ * @author kodrzywolek, Lukasz Warzecha
  */
 public class FlowClient {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FlowClient.class);
+
     private final Caller caller;
 
-    public FlowClient(String baseURL) {
-        caller = new Caller(baseURL);
+    private FlowClient(FlowClientBuilder builder) {
+        if (builder.username != null) {
+            this.caller = new Caller(builder.baseUrl, builder.username, builder.password, builder.connectionHelper);
+        } else {
+            this.caller = new Caller(builder.baseUrl);
+        }
     }
 
+    /**
+     * Creates new Voiceprint, which is represented by UUID.
```

Comment:
Proponuję (choć nie rozpisywałbym się tu za bardzo, dokładniejszy opis jest na swaggerze):

Creates a new Voiceprint. Returned ID should be passed to any subsequent operations on this Voiceprint (i.e. enrollment/verification).